packages/recomponents/src/components/r-toast/r-toast.story.js

```diff
@@ -4,12 +4,17 @@ import {action} from '@storybook/addon-actions';
 import markdown from './r-toast.md';
 import RToast from './r-toast.vue';
 
+function destroyToastManuallyWhenTheUserNavigatesToAnotherStory(className) {
+    document.querySelectorAll(className).forEach(el => el && el.remove());
+}
+
 storiesOf('Components.Toast', module)
     .addParameters({component: RToast})
     .add('Component', () => ({
         template: `
-            <div class="storybook-center">           
-                <r-toast :visible="true"
+            <div class="storybook-center">
+                <r-toast class="componentStory"
```

Comment:
how about class `component-story` instead of `componentStory`

Reply:
Ok. I'll change it.